simulation/README.md

```diff
@@ -44,8 +42,8 @@ Make sure you have some version of python installed on your personal computer. *
 
 ## Running the Simulation
 ### Docker Simulation Setup
-1. Open up a terminal and ssh into your Docker Container with the `-XY` flag (i.e. run `ssh -XY -p 2200 duke@[ip address]`).
-2. Run `source /opt/ros/kinetic/setup.bash` and `roscore &`.
+1. Open up a terminal and ssh into your Docker Container with the `-XY` flag (i.e. run `ssh -XY -p 2201 duke@[ip address]`).
```

Comment:
It should be root@localhost. That will always work for landside.

Reply:
Fixed.

Reply:
not quite, **root** at localhost, not duke

Reply:
Fixed for real, now.